EP.PowerShell.JiraDeployInfo/Public/Update-AzureDeploymentInformation.ps1

```diff
@@ -116,7 +116,7 @@ function Update-AzureDeploymentInformation {
         [string] $SystemAccessToken = $env:SYSTEM_ACCESSTOKEN,
 
         [Parameter(Mandatory=$false)][ValidateNotNullOrEmpty()]
-        [string] $AzureChangeUrl = "$env:SYSTEM_COLLECTIONURI/$env:SYSTEM_TEAMPROJECT/_traceability/runview/changes?currentRunId=$($env:BUILD_BUILDID)&__rt=fps"
+        [string] $AzureChangeUrl = "$env:SYSTEM_TASKDEFINITIONSURI/$env:SYSTEM_TEAMPROJECT/_traceability/runview/changes?currentRunId=$($env:BUILD_BUILDID)&__rt=fps"
```

Comment:
For some reason the COLLECTIONURI was prefixed with vsts, and this endpoint wasn't being found there. TASKDEFINITIONSURI just points to dev.azure.com, which does seem to have the API.

Reply:
Sounds like classic Microsoft renaming and rebranding, Visual Studio Team Services is now called: ..... etc.